content/en/blog/_posts/2024-secret-sync-controller-alpha.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: "Introducing Secret Sync Controller Alpha for Kubernetes "
+date: 2024-03-18
```

Comment:
Any particular time of day (and timezone perhaps) that you'd like this published?

Reply:
I think we would ideally aim for morning (perhaps around 9am) pacific time